pkg/topology/kademlia/kademlia.go

```diff
@@ -173,6 +181,12 @@ func New(
 	return k, nil
 }
 
+// produces oversaturation values using an exponential decay formula, sequence is as follows:
+// bin 0 -> 50, 44, 39, 34, 30, 27
+func defaultOverSaturationCalc(bin uint8) int {
+	return int(math.Round(50 * math.Exp(-float64(bin)/8.0)))
+}
+
```

Comment:
Should the 50 and the 8.0 be constants declared above?  Otherwise this completely hides the actual bin limits inside this function.